### ISSUE TITLE
GMFCS level II

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7279]
**Preferred term label:**
GMFCS level II

**Synonyms**
The Gross Motor Function Classification System level II

**Definition (free text, please give PubMed ID)**
Children with cerebral palsy classified under GMFCS level II can walk in most settings and climb stairs holding onto a railing; they may experience difficulty walking long distances and balancing on uneven terrain, inclines, in crowded areas or confined spaces; they may walk with physical assistance, a hand-held mobility device or use wheeled mobility over long distances; they have only minimal ability to perform gross motor skills such as running and jumping (PMID: 27188686).

**Parent term (use hpo.jax.org/app)**
Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
As mentioned, I think these terms are not good for HPO because they bundle a number of different items. For HPO, we want to capture the detailed data.
